seqr/views/apis/staff_api_tests.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+import json
+import mock
+
+ES_CAT_ALLOCATION=[{
+    u'node': u'node-1',
+    u'disk.used': u'67.2gb',
+    u'disk.avail': u'188.6gb',
+    u'disk.percent': u'26'
+  },
+  {u'node': u'UNASSIGNED',
+   u'disk.used': None,
+   u'disk.avail': None,
+   u'disk.percent': None
+  }]
+
+ES_CAT_INDICES=[{
+    "index": "test_index",
+    "docs.count": "122674997",
+    "store.size": "14.9gb",
+    "creation.date.string": "2019-11-04T19:33:47.522Z"
+  },
+  {
+    "index": "test_index_second",
+    "docs.count": "672312",
+    "store.size": "233.4mb",
+    "creation.date.string": "2019-10-03T19:53:53.846Z"
+  },
+  {
+    "index": "test_index_old",
+    "docs.count": "672312",
+    "store.size": "233.4mb",
+    "creation.date.string": "2019-10-03T19:53:53.846Z"
+  },
+  {
+    "index": "1kg.vcf.gz",
```

Comment:
this is a vcf name, never an index name. Please do not use it in the test data as that can be confusing

Reply:
The "1kg.vcf.gz" is from the fixture date 1kg_project.json. If it is improper, I will update the fixture data.

Reply:
I have changed it to "1kg_test_index". It will be an indice without active project.

Reply:
I decided not to change 1kg_project.json by removing `1kg.vcf.gz`. It is not required by this test case.